src/ruby/yast/convert.rb

```diff
@@ -37,7 +37,7 @@ module Convert
     # @!method  self.to_locale(        object )
     #   @return        [String, nil]  *object*, or `nil` if it is not a String
     Ops::SHORTCUT_TYPES.each do |type|
-      eval <<END
+      eval <<END, binding, __FILE__, __LINE__ + 1
```

Comment:
well, with my current ruby knowledge, I propose to remove eval and instead use `define_method`. How do you see it? https://apidock.com/ruby/Module/define_method